### ISSUE TITLE
[Website internal] Adopt Docsy blog, fix layout issues

### ISSUE THREAD

**Issue by chipzoller** [kyverno/website#565]
### Description

Docsy has a [blog capability](https://www.docsy.dev/blog/), which we'd like to use in the Kyverno website, but this probably requires an [update](https://github.com/kyverno/website/issues/564) of the theme. Although, as mentioned in that linked issue, an experimental update has been performed and a blog created, we have issues with partials when it comes to the banner, for example, as shown below.

<img width="1504" alt="image" src="https://user-images.githubusercontent.com/29008123/172964047-a677521c-7cd9-4e1b-a3d0-eddf7b30b190.png">

In addition, the creation of this banner causes slight other display issues when it comes to vertical scrolling, which exists today even without a Docsy update.

<img width="434" alt="image" src="https://user-images.githubusercontent.com/29008123/172964186-506ab2fc-d4d0-4618-a7c8-5307d72d2ff2.png">

In short, we'd like to adopt the Docsy theme's blog (which accompanies an update) and fix any display/layout related issues which may or may not be related to that.

**Comment by chalin** [comment]
> In addition, the creation of this banner causes slight other display issues when it comes to vertical scrolling, which exists today even without a Docsy update.

Are there separate issues open already that cover/report these problems?

**Comment by chipzoller** [comment]
No, but they're super minor. I can't even recall what they are right now.
